### ISSUE TITLE
doc/8.topics folder re-organization

### ISSUE THREAD

**Issue by Lucarda** [pure-data/pddp#18]
`doc/8.topics` should be prepared for a "better organization" as there might be more to come in this folder.

````
doc/
├── 8.topics/
    ├── slope~/
    ├── expr/
    ├── ...
````

**Comment by porres** [comment]
I would also maybe rename this folder, cause I don't understand what 'topic' means here

**Comment by danomatika** [comment]
Topic works for me:

> topic: a matter dealt with in a text, discourse, or conversation; a subject

ie. this section deal goes into detail on a few specific _topics_.

**Comment by umlaeute** [comment]
i guess the issue with "topic" that @porres is having, is that it is rather generic.
e.g. the reference patches (help-files) all deal with "a matter in a text, discourse, or conversation".
similarly, the "audio.examples" all form topics (e.g. "ring modulation").

at least *i* feel that pain. but i'm a non-native speaker.
if the term is clear enough for a native speaker, than that's fine for me (it's not that i don't have folders named "stuff" and similar for lack of a better term on my harddisk...)

**Comment by Lucarda** [comment]
For me "topic" was always good. We have in spanish "el tópico de la conversación fue ..."

**Comment by porres** [comment]
> i guess the issue with "topic" that @porres is having, is that it is rather generic.
> e.g. the reference patches (help-files) all deal with "a matter in a text, discourse, or conversation".
> similarly, the "audio.examples" all form topics (e.g. "ring modulation").

I feel it is too generic and does not clarify its purpose. For now, what we have is just 2 html files that are detailed references of two objects "expr" and "slop~"

If we had html references for all help files in here, or if we end up doing it, this would be the place to put it and it could be renamed to 'html-reference' or something... but we do not have yet an actual plan to expand this. The discussion here seems to lead that we don't need this as much => https://github.com/pure-data/pddp/issues/10

As far as 'reference' goes, as an accompaniment to the help files, I'm proposing we add subpatches, see https://github.com/pure-data/pddp/issues/6 (I'm already on it by the way). 

Now, it's stupid to rename 'topics' to 'html reference' when we already have a folder called 'reference', even though this folder just has help files. But what if we moved the contents of '8.topic' to a subfolder in '5.reference'? :) It would make total sense for the contents it has so far and it'd be easier to call them in the help files. I'm thinking of 'html.reference' subfolder, and they can be further reorganized into 'expr' and d'slop~' subfolders.

8.topics would be then removed, unless we actually want such a folder for 'something', which is no yet clear to me what it could be...

**Comment by danomatika** [comment]
I would leave it as is since I don't consider it a major issue worth the extra energy. Besides, all of this is academic if Miller prefers his layout (which I believe he does).

**Comment by Lucarda** [comment]
I agree with @danomatika . eventually if we need more folders for html stuff it can go elsewhere in `doc` or on-line-only.

Feel free to reopen.

**Comment by porres** [comment]
before closing for good, I'd like to hear @millerpuckette 's thoughts on this and to clear out what's the idea behind this 'topics' folder.

**Comment by porres** [comment]
I guess we can forget about the need to rename 'topics' to something else and stuff, but I'd like to organize this and put all things from 'slop~' into its own folder...
